jwala-common/src/main/java/com/cerner/jwala/common/rule/webserver/WebServerNameRule.java

```diff
@@ -19,4 +19,10 @@ protected MessageResponseStatus getMessageResponseStatus() {
     protected String getMessage() {
         return "Invalid WebServer Name : \"" + name + "\"";
     }
+
+    @Override
+    public boolean isValid() {
+        return name != null && !"".equals(name.trim()) && name.matches("[A-Za-z0-9._\\s-]+");
```

Comment:
Is a name with a leading or trailing space really correct? Should we allow that?